hazelcast/src/main/java/com/hazelcast/internal/networking/nio/NioNetworking.java

```diff
@@ -102,7 +103,7 @@
     private volatile NioThread[] inputThreads;
     private volatile NioThread[] outputThreads;
 
-    // Currently this is a course grained aggregation of the bytes/send reveived.
+    // Currently this is a course grained aggregation of the bytes/send received.
```

Comment:
While we're at it: `course` -> `coarse`

Reply:
Thanks for finding this typo! I'll fix this one.